exercises/csharp/TriviaGame/TriviaGame/GoldenMaster.cs

```diff
@@ -16,6 +16,7 @@ public static string GetResult(int i)
             }
             catch (IOException e)
             {
+				Console.WriteLine(e);
```

Comment:
It's good that the error is being logged in the console so we know what went wrong.